tests/ops/test_stats.py

```diff
@@ -37,7 +115,7 @@ def test_cummin():
 
 @pytest.mark.parametrize('statistics', [autocorrelation, autocovariance, _cummin])
 @pytest.mark.parametrize('sample_shape', [(), (3,), (2, 3)])
-def test_statistics_ok_with_sample_shape(statistics, sample_shape):
+def test_statistics_B_ok_with_sample_shape(statistics, sample_shape):
```

Comment:
Not for this PR, but `test_effective_sample_size` gives 107.85 which is very different from the expected value of 134.5. I remember you had a fix for CPU, but I am not sure why that's not working for CUDA.

Reply:
It might be that cuda uses a different fft library. I'll try to find some cuda resources to test it.

Reply:
I see; I think we can defer it to later.